### PR TITLE
Add NetworkPolicy for CockroachDB

### PR DESCRIPTION
**Pull request by ozdanborne**
cc @lachie83

### CODE REVIEW

**Review by lachie83 — APPROVED** [review]

**Comment by lachie83** [comment]
cc @a-robinson

File: stable/cockroachdb/README.md [hunk 1]
```diff
@@ -178,6 +180,22 @@ nodeID:     2
...
+install [a networking plugin that implements the Kubernetes
+NetworkPolicy spec](https://kubernetes.io/docs/tasks/administer-cluster/declare-network-policy#before-you-begin),
+and set `NetworkPolicy.Enabled` to `true`.
+
+For Kubernetes v1.5 & v1.6, you must also turn on NetworkPolicy by setting
+the DefaultDeny namespace annotation. Note: this will enforce policy for _all_ pods in the namespace:
+
+    kubectl annotate namespace default "net.beta.kubernetes.io/network-policy={\"ingress\":{\"isolation\":\"DefaultDeny\"}}"
+
+For more precise policy, set `networkPolicy.allowExternal=true`. This will
```
Comment (a-robinson) [inline stable/cockroachdb/README.md]: Shouldn't this say to set it to `false`?

**Review by a-robinson — APPROVED** [review]
LGTM, thanks @ozdanborne!

Is this something that you're using or are you just going around spreading the NetworkPolicy love?

**Comment by ozdanborne** [comment]
@a-robinson just spreading the love. Open to feedback, especially from actual users.

**Comment by ozdanborne** [comment]
@a-robinson how are we looking? Tests are passing :smile:

**Review by a-robinson — APPROVED** [review]
LGTM. I didn't realize it was being held up by me, sorry!

**Pull request merged**